### DISCUSSION TITLE
Help me sell in Payload... Who is using it in the real world?

### DISCUSSION THREAD

**Discussion by gekkota** [payloadcms/payload#2483] [General]
Hi All,

I've been playing round with, and really enjoying Payload CMS. Im really keen for work to adopt it going forwards, but as always the people above want some kind of reassuring noises.

I was wondering if there are any well known companies out there using Payload, I appreciate it doesn't really matter, but you know how these things go.

Aside easy of use and speed of development, what other good points should I highlight?

Really appreciate your help, thanks

**Comment by denolfe** [comment]
Hey @gekkota, I definitely understand the need for assurances when trying to encourage the adoption of a new piece of technology.

We can't name any of our clients (yet), but we do have a Fortune 20 company building with Payload, along with a few other Fortunate 500 companies. We'll be working on a campaign in the near future to be able to surface this on our website to make pitching Payload much easier.

Also, through our _completely anonymized analytics_, there are over 8,000 projects currently running Payload in production.
